src/pages/workspaces/workspace/analysis/runtime-utils.js

```diff
@@ -506,3 +508,17 @@ export const cloudProviders = {
 export const isGcpContext = ({ cloudProvider }) => cloudProvider === cloudProviders.gcp.label
 export const isAzureContext = ({ cloudProvider }) => cloudProvider === cloudProviders.azure.label
 
+//TODO: fields isAppStatus? LeoLabel? isRuntimeStatus?
+export const runtimeStatuses = {
```

Comment:
Nit: Thoughts on 'value' instead of 'leoLabel'?

Reply:
Value is pretty ambiguous. There's 2 `value`s at play here, one being the string leo returns to represent a status, the other being the string the UI displays to users to represent a status.